lib/rubocop/cop/style/nested_function_definition.rb

```diff
@@ -0,0 +1,19 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for nested function definitions.
```

Comment:
I'd expand the comment here with an explanation of the problem and code snippets.